_test/test_sqw_file/test_faccess_sqw_prototype.m

```diff
@@ -23,104 +23,104 @@
             co = onCleanup(@()to.delete());
-            
+
             warning('off','SQW_FILE_IO:legacy_data')
             this.clob = onCleanup(@()(warning('on','SQW_FILE_IO:legacy_data')));
-            
+
             [ok,inob] = to.should_load(obj.sample_file);
             co1 = onCleanup(@()(fclose(inob.file_id)));
-            
+
             assertTrue(ok);
             assertTrue(inob.file_id>0);
-            
+
         end
-        
+
         function obj = test_init(obj)
             to = faccess_sqw_prototype();
             assertEqual(to.file_version,'-v0');
-            
+
             %access to incorrect object
             f = @()(to.init());
             assertExceptionThrown(f,'SQW_FILE_IO:invalid_argument');
-            
+
             warning('off','SQW_FILE_IO:legacy_data')
             this.clob = onCleanup(@()(warning('on','SQW_FILE_IO:legacy_data')));
-            
+
             [ok,inob] = to.should_load(obj.sample_file);
-            
+
             assertTrue(ok);
             assertTrue(inob.file_id>0);
-            
+
             to = to.init(inob);
             assertEqual(to.npixels,16);
-            
+
             header = to.get_header();
             assertEqual(header.filename,'map11014.spe')
             assertEqual(header.ulabel{4},'E')
             assertEqual(header.ulabel{3},'Q_\eta')
-            
+
             det = to.get_detpar();
             assertEqual(det.filename,'demo_par.PAR')
             assertEqual(det.filepath,'d:\users\abuts\SVN\ISIS\HoraceV1.0final\documentation\')
             assertEqual(numel(det.group),28160)
-            
+
             data = to.get_data();
             assertEqual(size(data.pix),[9,16])
             assertEqual(size(data.s,1),numel(data.p{1})-1)
             assertEqual(size(data.e,2),numel(data.p{2})-1)
             assertEqual(size(data.npix,3),numel(data.p{3})-1)
-            
+
         end
         function obj = test_get_data(obj)
             %spath = fileparts(obj.sample_file);
             warning('off','SQW_FILE_IO:legacy_data')
             this.clob = onCleanup(@()(warning('on','SQW_FILE_IO:legacy_data')));
-            
+
             to = faccess_sqw_prototype(obj.sample_file);
-            
+
             data_h = to.get_data('-he');
             assertTrue(isstruct(data_h))
             assertEqual(data_h.filename,to.filename)
             assertEqual(data_h.filepath,to.filepath)
-            
+
             data_dnd = to.get_data('-ver','-nopix');
             assertTrue(isa(data_dnd,'data_sqw_dnd'));
             assertEqual(data_dnd.filename,'test_sqw_read_write_v0_t.sqw');
-            
-            data = to.get_data('-ver',1,10);
+
+            data = to.get_data('-ver');
             assertEqual(data.filename,data_dnd.filename)
             assertEqual(data.filepath,data_dnd.filepath)
-            assertEqual(size(data.pix),[9,10]);
+            assertTrue(isa(data.pix, 'PixelData'));

```

Comment:
should this PixelData class contain any data? 
Would an additional `data.pix.num_pixels` tests be useful?